### PR TITLE
feat(presentational): add Pagination component 

### PR DESCRIPTION
**Pull request by agory**
This pagination component uses the style of the Bulma [pagination](https://bulma.io/documentation/components/pagination/) component.

### CODE REVIEW

File: src/web/components/Pagination/Pagination.test.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Pagination from './';
+import Button from '../Button';
+
+describe('Pagination', () => {
```
Comment (mfrachet) [inline src/web/components/Pagination/Pagination.test.js]: It seems that some limit test are missing:

- What if I'm on page 1 and that I try to click `select page - 1`
- What if I'm on a `maxPage` (like 100) that I try to click on `select page + 1`

😊

File: src/web/components/Pagination/index.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import COLOR from '../../constant';
+import Button from '../Button';
+
+class Pagination extends React.Component {
+  constructor(props) {
+    super(props);
+    this.nextPageHandler = () => {
+      this.props.onChange(this.props.currentPage + 1);
```
Comment (mfrachet) [inline src/web/components/Pagination/index.js]: You can destructure in all the function your `props` to create something more readable 😊

File: src/web/components/Pagination/index.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    };
+
+    this.previousPageHandler = () => {
```
Comment (mfrachet) [inline src/web/components/Pagination/index.js]: `this`keyword is not mandatory I think 🤔

File: src/web/components/Pagination/index.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import COLOR from '../../constant';
+import Button from '../Button';
+
+class Pagination extends React.Component {
+  constructor(props) {
+    super(props);
+    this.nextPageHandler = () => {
```
Comment (mfrachet) [inline src/web/components/Pagination/index.js]: Why to declare your method inside the `constructor` ?

File: src/web/components/Pagination/index.js [hunk 5]
```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import COLOR from '../../constant';
+import Button from '../Button';
+
+class Pagination extends React.Component {
```
Comment (mfrachet) [inline src/web/components/Pagination/index.js]: Components should be max 100 lines of code 😝 . Challenging right ? 😎

**Comment by agory** [comment]
need #38

**Comment by agory** [comment]
I don't know if the challenge is successful but I've tried  :smile:

**Comment by mfrachet** [comment]
Nearly good ! 😝 

I've tried to play with the storybook, but the pages don't change 🤔 

Maybe you can try to set an internal state to the pagination, like `currentPage`, and add a particular style when this page change.

Also, keep the `onChange` method available and exposed for the parent component 😊

**Comment by mfrachet** [comment]
When I click a specific pagination, I'm on page n+1 

Try to select page 5 ;)

**Comment by agory** [comment]
Are you talking about the blue effect related to the focus of the button?

**Comment by mfrachet** [comment]
I'm talking about the fact that when I click on the 5 page, I'm on the 6 one 😛

**Comment by agory** [comment]
I have removed the focus border and change default color. For the moment, I chose the color dark however it could be changed later for more homogeneity

**Comment by agory** [comment]
I have update jest configuration with https://facebook.github.io/jest/docs/en/webpack.html to mock css import (bug with pagination css to custom buttun:focus)

**Pull request merged**